Gemfile

```diff
@@ -0,0 +1,34 @@
+source 'https://rubygems.org'
+
+gem 'rails',                     '4.2.2'
```

Comment:
maybe, for an example, it should be used the newest version (actual 4.2.5.1),
or better (IMHO) without fixing the version
(same for the other gems)